x-pack/plugins/ml/public/components/influencers_list/_influencers_list.scss

```diff
@@ -4,6 +4,23 @@
   .field-label {
     font-size: $euiFontSizeXS;
     text-align: left;
+    max-height: $euiFontSizeS;
```

Comment:
In combination with the other suggestion of moving `EntityCell` to the general components folder, could this whole section of SCSS be part of the `EntityCell` component?

Reply:
Fixed in: 47d51d2810eb15b7497e6b5af8b5094b18db0280